tests/tensorflow/nightly/ncf.libsonnet

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local common = import "common.libsonnet";
+local mixins = import "templates/mixins.libsonnet";
+local tpus = import "templates/tpus.libsonnet";
+local gpus = import "templates/gpus.libsonnet";
+
+{
+  local ncf = common.ModelGardenTest {
+    modelName: "ncf",
+    command: [
+      "python3",
+      "official/recommendation/ncf_keras_main.py",
```

Comment:
does this test do any assert on accuracy?  Otherwise your tests won't send any alerts if convergence accuracy gets worse

Reply:
Added check for `hit_rate` metric to be at least 0.62 in convergence tests:

    regressionTestConfig+: {
      metric_success_conditions+: {
        "hit_rate": {
          success_threshold: {
            fixed_value: 0.62,
          },
          comparison: "greater",
        },
      },
    },